src/Sylius/Bundle/RbacBundle/spec/Form/Type/PermissionTypeSpec.php

```diff
@@ -46,17 +47,18 @@ function it_should_build_form_with_proper_fields(FormBuilder $builder)
         ;
 
         $builder
-            ->add('parent', 'sylius_permission_choice', Argument::any())
+            ->addEventSubscriber(Argument::type(AddCodeFormSubscriber::class))
             ->shouldBeCalled()
```

Comment:
Removing it will force us to add them in defined previously order, as only the last call needs to be checked against. However, this can lead us to weird behaviour, as removing some middle calls in the SUT won't result in any error.